src/Collapse/CollapseGroup/CollapseGroup.component.spec.tsx

```diff
@@ -0,0 +1,56 @@
+// REACT
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+// Vendor
+import { shallow, mount } from 'enzyme';
+// COMPONENT
+import { CollapseGroup } from './CollapseGroup.component';
+import { Collapse } from '../Collapse.component';
+// import { ArrowBack } from '../../Icon';
+
+describe('Component: CollapseGroup', () => {
+    it('should match its snapshot.', () => {
+        const subject = <CollapseGroup />;
+        const component = shallow(subject);
+
+        expect(subject).toBeDefined();
+        expect(component).toBeDefined();
+
+        const tree = renderer.create(subject).toJSON();
+
+        expect(tree).toMatchSnapshot();
+    });
+
+    it("should show only one Collapse component's content when accordion is true.", () => {
+        const subject = (
+            <CollapseGroup accordion openIndex={1}>
+                <Collapse>Hello World</Collapse>
+                <Collapse>Hello World</Collapse>
+                <Collapse>Hello World</Collapse>
+            </CollapseGroup>
+        );
+        const wrapper = mount(subject);
+        const toggleButton = wrapper.find('.anchor-collapse-button').first();
```

Comment:
Is this simulating clicking the `Collapse` that is already open? I just want to make sure this is testing the functionality of clicking a different `Collapse` and _then_ making sure there is still only 1 open.

Reply:
It simulates clicking a closed one. The `openIndex` prop that is passed makes it so the second Collapse child is open, then the simulated click is on the first one which is closed.